src/utils/common.ts

```diff
@@ -360,3 +361,28 @@ export function getNumberOfDaysInMonth(year: number, month: number) {
     const dateWithLastDateOfPrevMonth = new Date(year, month + 1, 0);
     return dateWithLastDateOfPrevMonth.getDate();
 }
+
+export function joinList(list: React.ReactNode[], separator: React.ReactNode) {
```

Comment:
is this moved from somewhere?

Reply:
Nope, brand new stuff

Reply:
Need to check if this stuff works :1234:

Reply:
How are we defining keys here